Makefile

```diff
@@ -15,7 +15,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.22
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.22.8
```

Comment:
Is there a need to pin to the patch?  This will automatically use the most recent version of `1.22`.

Reply:
Reverted this change!